### DISCUSSION TITLE
Github widgets for android

### DISCUSSION THREAD

**Discussion by aayushchugh** [community/community#27261] [Mobile] [answered]
It would be great if we have github mobile widgets on android
eg- 
 - contribution graph
 - recent issues/ assigned issues

**Comment by NitinIsFit** [comment]
I totally agree

**Comment by aconsuegra** [comment, marked as answer]
Hi @aayushchugh, thanks for getting in touch and the feature request! I'll bring it to our Product team so we can prioritize it accordingly.

**Reply by aayushchugh** [reply]
Thanks for the reply. I'm waiting to see this feature in future releases
